examples/metalk8s-solution-example/operator/pkg/controller/util/baseserver.go

```diff
@@ -109,9 +110,12 @@ func BuildLabelSelector(labels map[string]string) metav1.LabelSelector {
 // buildImageName builds a complete image name based on the version provided
 // for the `base-server` component, which is the only one deployed for now
 func buildImageName(version string) string {
-	prefix, found := os.LookupEnv("REGISTRY_PREFIX")
-	if !found {
-		prefix = "docker.io/metalk8s"
+	var prefix string = "docker.io/metalk8s"
```

Comment:
Let's define this as a default value for the config and only read from config in controllers.

See https://github.com/spf13/viper#establishing-defaults